index.html

```diff
@@ -320,11 +320,77 @@ <h2 class="food-spin__location-title js-food-spin-location-title">Pick Up </h2>
         </svg>
       </button>
     </section>
+      <div class="cart-summary js-cart-summary" data-open="true">
+        <div class="cart-summary__top">
+          <h2 class="cart-summary__title">My Shopping Cart</h2>
+            <button class="cart-summary__close js-cart-summary-close">
+              <svg class="cart-summary__svg" aria-label="enter store" width="18" height="18" viewBox="0 0 28 29" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M0 29L12.631 13.34L1.93862 0H4.59672L14.01 11.6879L23.4033 0H26.0614L15.369 13.34L28 29H25.262L14.01 15.0273L2.73804 29H0Z" />
+              </svg>
+            </button>
+        </div>
+        <div class="cart-summary__content">
+          <div class="cart-summary__item">
+            <div class="cart-summary__description">
+              <p class="cart-summary__name js-cart-summary-name">Green Goddess Chicken Salad</p>
+              <span class="cart-summary__prics js-cart-summary-prics">$32.99</span>
+            </div>
+            <div class="cart-summary__discount">
+              <span class="cart-summary__discount-description">Discount Applied From:</span>
+              <span class="cart-summary__discount-price js-cart-summary-discount-price">-$37.00</span>
+            </div>
+            <a class="cart-summary__discount-price__remove" href="#" class="remove-link">Remove</a>
```

Comment:
En BEM no se usan dos `__` en un nombre de clase
```suggestion
            <a class="cart-summary__remove-dish" href="#" class="remove-link">Remove</a>
```